rio/subsystems/drivesubsystem.py

```diff
@@ -66,7 +84,7 @@ def __init__(self) -> None:
         self.leftEncoder.setDistancePerPulse(encoderDistPerP)
         self.rightEncoder.setDistancePerPulse(encoderDistPerP)
 
-        self.gyro = wpilib.ADXRS450_Gyro()
+        self.gyro = wpilib.ADXRS450_Gyro()  # We may need to use another gyro
```

Comment:
This gyro comes from the boilerplate code, consider using the [pigeon from last year](https://github.com/FRC-1721/1721-RapidReact/blob/a8ba43fd3ed713b8fc7e0688153e1a2970b03d00/rio/subsystems/drivetrain.py#L58-L68)